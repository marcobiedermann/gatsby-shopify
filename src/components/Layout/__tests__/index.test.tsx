import { render } from '@testing-library/react';
import React from 'react';
import { Layout } from '..';

describe('layout component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <Layout
        shopifyShop={{
          name: 'ShopifyShop',
        }}
      >
        Content
      </Layout>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
