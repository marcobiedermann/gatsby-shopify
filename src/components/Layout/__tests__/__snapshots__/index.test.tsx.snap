// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`layout component should render correctly 1`] = `
<DocumentFragment>
  <div
    class="layout"
  >
    <header
      class="header"
    >
      <a
        href="/"
      >
        ShopifyShop
      </a>
      <nav
        class="navigation"
      >
        <ul
          class="navigation__list navigation__listInline"
        >
          <li>
            <a
              class="navigation__link"
              href="/collections"
            >
              Collections
            </a>
          </li>
          <li>
            <a
              class="navigation__link"
              href="/products"
            >
              Products
            </a>
          </li>
          <li>
            <a
              class="navigation__link"
              href="/blogs/news"
            >
              News
            </a>
          </li>
        </ul>
      </nav>
    </header>
    <main
      class="main"
    >
      Content
    </main>
    <footer
      class="footer"
    >
      <div
        class="grid"
      >
        <nav
          class="navigation"
        >
          <ul
            class="navigation__list"
          >
            <li>
              <a
                class="navigation__link"
                href="/policies/privacy-policy"
              >
                Privacy Policy
              </a>
            </li>
            <li>
              <a
                class="navigation__link"
                href="/policies/refund-policy"
              >
                Refund Policy
              </a>
            </li>
            <li>
              <a
                class="navigation__link"
                href="/policies/terms-of-service"
              >
                Terms of Service
              </a>
            </li>
          </ul>
        </nav>
      </div>
    </footer>
  </div>
</DocumentFragment>
`;
